packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -295,11 +297,11 @@ export class Meter implements api.Meter {
    * each aggregator belonging to the metrics that were created with this
    * meter instance.
    */
-  async collect(): Promise<void> {
+  collect(): Promise<void> {
```

Comment:
Why did you remove async?

Reply:
because it is returning promise.all anyway , no need to add extra generators